bftengine/tests/simpleKVBCTests/TesterClient/main.cpp

```diff
@@ -28,24 +31,39 @@
 #include "winUtils.h"
 #endif
 
+#ifdef USE_LOG4CPP
+#include <log4cplus/configurator.h>
+#endif
+
 using namespace SimpleKVBC;
+using namespace bftEngine;
 
 using std::string;
-																						 
+
+concordlogger::Logger clientLogger =
+		concordlogger::Logger::getLogger("skvbctest.client");
+
 int main(int argc, char **argv) {
 #if defined(_WIN32)
 	initWinSock();
 #endif
 
-	char argTempBuffer[PATH_MAX + 10];
+#ifdef USE_LOG4CPP
+	using namespace log4cplus;
+  initialize();
+  BasicConfigurator logConfig;
+  logConfig.configure();
+#endif
 
-	uint16_t clientId = UINT16_MAX;
-	uint16_t fVal = UINT16_MAX;
-	uint16_t cVal = UINT16_MAX;
-	uint32_t numOfOps = UINT32_MAX;
+	char argTempBuffer[PATH_MAX + 10];
+	ClientParams cp;
+	cp.clientId = UINT16_MAX;
+	cp.numOfFaulty = UINT16_MAX;
+	cp.numOfSlow = UINT16_MAX;
+	cp.numOfOperations = UINT16_MAX;
 
 	int o = 0;
-	while ((o = getopt(argc, argv, "i:f:c:p:")) != EOF) {
+	while ((o = getopt(argc, argv, "i:f:c:p:n:")) != EOF) {

```

Comment:
Why are f and c still needed for the client process? Shouldn't these come from the config file now?

Reply:
Today f and c are defined from replica keys configuration file and the client doesn't need it.
When we will switch to full configuration file for the concord nodes - these parameters will be cleaned from the CLI.

Reply:
Today f and c are defined from replica keys configuration file and the client doesn't need it.
When we will switch to full configuration file for the concord nodes - these parameters will be cleaned from the CLI.